addon/templates/components/polaris-empty-search-result.hbs

```diff
@@ -0,0 +1,25 @@
+{{#polaris-stack
+  alignment="center"
+  vertical=true
+}}
+  {{#if withIllustration}}
+    {{!-- TODO: replace this with polaris-image when implemented. --}}
+    <img
+      src="empty-search.svg"
+      alt="Empty search results"
+      class="Polaris-EmptySearchResult__Image"
+    >
```

Comment:
Any reason for not having the [`draggable`](https://github.com/Shopify/polaris-react/blob/master/src/components/EmptySearchResult/EmptySearchResult.tsx#L41) attribute on this?

Reply:
Nope just missed it 😳